.github/workflows/verify-images.yaml

```diff
@@ -0,0 +1,25 @@
+name: Verify Docker Compose Images
+
+on:
+  push:
+    paths:
+      - '**.yaml'
+
+jobs:
+  verifyImages:
+    strategy:
+      matrix:
+        go-version: [1.18.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
```

Comment:
This can change to `ubuntu-latest` then we can remove the os / matrix parts.

Reply:
Done